depthai_sdk/src/depthai_sdk/oak_outputs/xout/xout_frames.py

```diff
@@ -29,74 +26,33 @@ def __init__(self, frames: StreamXout, fps: float = 30, frame_shape: Tuple[int,
         """
         self.frames = frames
         self.name = frames.name
-
-        self.fps = fps
```

Comment:
Ideally, we wouldn't specify FPS, as it can depend on the throughput, and it's not always 30 even if you set it to 30. In `AvWriter` we now save without knowing FPS upfront, we just specify timestamps. I'll experiment around with pyav for unencoded streams as well.